.github/workflows/block-performance.yml

```diff
@@ -54,40 +54,13 @@ jobs:
           mkdir results
           pnpm install
           pnpm config:decrypt
-          pnpm env:start
-
           pnpm tunnel:on
-          pnpx jetpack docker --type e2e --name t1 wp plugin deactivate jetpack
-          pnpx jetpack docker --type e2e --name t1 wp -- user create admin admin@example.com --role=administrator --user_pass=password
-
-      - name: Run performance base test
-        working-directory: gutenberg
-        run: |
-          export WP_BASE_URL=$(cat ../tools/e2e-commons/config/tmp/e2e-tunnels.txt)
-
-          npm run test-performance packages/e2e-tests/specs/performance/post-editor.test.js
-          mv packages/e2e-tests/specs/performance/post-editor.test.results.json ../tools/e2e-commons/results/base.test.results.json
-
-      - name: Environment reset
-        working-directory: tools/e2e-commons
-        run: |
-          pnpm env:reset
-          pnpm tunnel:reset
+          pnpm env:start
```

Comment:
> The redundant commands are tunnel:on and env:start

We still need these. Since the script itself does not start the containers nor tunnel. Not sure about tunnels, but `env:reset` won't work without `env:start`.

Reply:
I removed `tunnel:on`. but `env:start` is needed anyways.

>  setup part can be moved inside the script too

I'm not sure about that. I'd prefer having it separate. at lest for now.